test/uxCapture.test.js

```diff
@@ -84,6 +78,16 @@ describe('UXCapture', () => {
 
 			expect(onMeasure).not.toHaveBeenCalled();
 		});
+
+		it('should throw an error if called more than once, but without startTransition', () => {
+			UXCapture.startView([]);
+
+			expect(() => {
+				UXCapture.startView([]);
+			}).toThrowError(
+				'[UX Capture] Application should call UXCapture.startTransition() before starting new view'
```

Comment:
I wanted to listen to specific message because errors can be thrown with poor implementation and I wanted to be a bit more specific but without creating our own class of errors.

+1 on using a constant though.